boefjes/Dockerfile

```diff
@@ -30,6 +30,8 @@ RUN --mount=type=cache,target=/root/.cache \
     && rm /tmp/boefjes-requirements.txt \
     && if [ "$ENVIRONMENT" = "dev" ]; then pip install -r requirements-dev.txt; fi
 
+FROM dev
+
 COPY octopoes/ /tmp/octopoes
 RUN cd /tmp/octopoes && python setup.py bdist_wheel
 RUN pip install /tmp/octopoes/dist/octopoes*.whl
```

Comment:
This means that the development containers do not have octopoes installed as a package in their environment anymore. (It still works since we mount octopoes at the $PYTHONPATH.) This might expose us potential issues whe e.g. octopoes introduces new dependencies in `install_requires` or other setup changes.

Reply:
It is indeed a leftover, but if we choose this path perhaps we should remove the setup.py etc. as well? It's mainly the inconsistency that might introduce a small risk over a small performance gain in this case.

Reply:
I think we shouldn't remove setup.py because we currently still install octopoes in the container images and Debian packaging and changing this should probably be done as part of a bigger reorganization. But we can remove the install_required from setup.py, we don't really use it at the moment and that would get rid of the inconsistency.

Reply:
Created #807 to do this and clean up `setup.py` across modules.